javascript/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# syntax=docker.io/docker/dockerfile:1.4
+
+# build stage: includes resources necessary for installing dependencies
+
+# Here the image's platform does not necessarily have to be riscv64.
+# If any needed dependencies rely on native binaries, you must use 
+# a riscv64 image such as cartesi/node:19-jammy for the build stage,
+# to ensure that the appropriate binaries will be generated.
+FROM node:19-bullseye  as build-stage
+
+WORKDIR /opt/cartesi/dapp
+COPY . .
+RUN yarn
+RUN yarn build
```

Comment:
We can run the commands in a single run section to reduce the image layer (Even in a base image)

```sh

RUN yarn && \
         yarn build

```